waiter/src/waiter/core.clj

```diff
@@ -921,8 +924,14 @@
    :token->token-metadata (pc/fnk [[:curator kv-store]]
                             (fn token->token-metadata [token]
                               (sd/token->token-metadata kv-store token :error-on-missing false)))
-   :validate-service-description-fn (pc/fnk [[:state service-description-builder]]
+   :validate-service-description-fn (pc/fnk [[:state authenticator service-description-builder]]
                                       (fn validate-service-description [service-description]
+                                        (let [authentication (or (get service-description "authentication") "standard")
```

Comment:
This `let` block and its two locals can move outside the `fn` definition.

Reply:
done